man/bemenu.1.in

```diff
@@ -0,0 +1,311 @@
+.Dd March 29, 2021
+.Dt BEMENU 1
+.Os
+.Sh NAME
+.Nm bemenu
+.Nd dynamic menu inspired by
+.Xr dmenu 1
+.Sh SYNOPSIS
+.Nm
+.Op Fl hivwx
+.Op Fl I Ar index
+.Op Fl l Ar lines
+.Op Fl P Ar prefix
+.Op Fl p Ar prompt
+.Op Fl -ifne
+.Op Fl -scrollbar Cm none | always | autohide
+.Op Ar backend-options
+.Nm bemenu-run
+.Op Fl hivwx
+.Op Fl I Ar index
+.Op Fl l Ar lines
+.Op Fl P Ar prefix
+.Op Fl p Ar prompt
+.Op Fl -ifne
+.Op Fl -scrollbar Cm none | always | autohide
+.Op Fl -fork
+.Op Fl -no-exec
+.Op Ar backend-options
+.Sh DESCRIPTION
+.Nm
+is a dynamic menu for
+.Xr tty 4
+.Pq using Xr ncurses 3 ,
+X11 and Wayland, inspired by
+.Xr dmenu 1 .
+It reads a list of newline-separated items from standard input and then
+presents them as a list, where the user can select one or more of them.
+When pressing
+.Aq Cm carriage-return ,
+the selected items are printed to standard output (one per line) and
+.Nm
+exits.
+Entering text will filter the items to those that match the input.
+If the number of items exceeds the size of the list,
+the items will be paginated.
+.Pp
+.Nm bemenu-run
+is a special-case invocation of
+.Nm ,
+where the input is the list of executables under
+.Ev PATH
+and the selected items are executed.
+.Pp
+The options are as follows:
+.Bl -tag -width Ds
+.It Fl h , -help
+Print a help message to standard output and exit.
+.It Fl I Ar index , Fl -index Ar index
+Select the item at
+.Ar index
+at the start.
+Indices start at 0.
+.It Fl i , -ignorecase
+Filter items case-insensitively.
+.It Fl l Ar number , Fl -list Ar number
+List items vertically with the given
+.Ar number
+of lines.
+.It Fl P Ar prefix , Fl -prefix Ar prefix
+Display
+.Ar prefix
+before the highlighed item in a vertical list.
+.It Fl p Ar prompt , Fl -prompt Ar prompt
+Defines the
+.Ar prompt
+text to be displayed to the left of the input field.
+Defaults to
+.Ql bemenu
+.It Fl v , -version
+Print the
+.Nm
+version number to standard output and exit.
+.It Fl w , -wrap
+Wraps cursor selection.
```

Comment:
I tried testing this and I still don't know what this does, so I left it as is.

Reply:
The `-w` option should make the list go back to first item once you go one over the end. Thus "wrapping around".